### PR TITLE
CNV-41205: fix sameVolume func

### PR DESCRIPTION
**Pull request by upalatucci**
## 📝 Description

small fix

### CODE REVIEW

File: src/views/virtualmachines/actions/components/DeleteVMModal/utils/helpers.ts [hunk 1]
```diff
@@ -69,4 +69,4 @@ export const removeDataVolumeTemplatesToVM = (
 export const sameVolume = (
   volumeA: IoK8sApiCoreV1PersistentVolumeClaim | V1beta1DataVolume,
   volumeB: IoK8sApiCoreV1PersistentVolumeClaim | V1beta1DataVolume,
-) => volumeA.kind === volumeB.kind && getName(volumeA) === volumeB;
+) => volumeA.kind === volumeB.kind && getName(volumeA) === getName(volumeB);
```
Comment (avivtur) [inline src/views/virtualmachines/actions/components/DeleteVMModal/utils/helpers.ts:72]: We should also compare namespaces here

Comment (avivtur) [reply src/views/virtualmachines/actions/components/DeleteVMModal/utils/helpers.ts:72]: And can't we use isEqualObject function here? why is this function needed?

Comment (upalatucci) [reply src/views/virtualmachines/actions/components/DeleteVMModal/utils/helpers.ts:72]: I would not compare them with isEqualObject as a small change can result in a false negative. Maybe one is an older version of the other or something like that

Comment (upalatucci) [reply src/views/virtualmachines/actions/components/DeleteVMModal/utils/helpers.ts]: done

**Review by metalice — APPROVED** [review]

**Pull request merged**